### ISSUE TITLE
implement "Omaha High/Low (x or better)"

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#1094]
Implement "Omaha High/Low (x or better)":
> each hand after the flop betting 2 dice are rolled to determine low qualifier (e.g., 2-4 = high only, 5 = wheel only, 11 = J or better, 12 = Q or better)

I've labeled this as a "request", because I view it as an implicit request. I figure people might like to practice the game before it's spread at EMBARGO. I've labeled this "high priority", because I think I can crank it out today. However, if I run into something unexpected, I might not implement it at all. I have a lot of non-programming to do and I don't want to pause it very long.

**Comment by ctm** [comment]
FWIW, to do this the "right way" would take way too long, because it would require me to refactor the code so that game-specific state is associated with the game, rather than the table. However, Binglaha is already implemented with a kludge, that gives `Table` a `die` field that is currently an `Option<SixSidedDie>`. So, the hack-and-slash way to proceed is to turn that into a `Vec<SixSidedDie>` and the have zero elements correspond to `None`, use one element for Binglaha and two elements for the `or better` variant.

If God is a Rust programmer, I'm going to hell, [but at least I'm enjoying the ride](https://www.youtube.com/watch?v=a9Bs4xhDyxw).

**Comment by ctm** [comment]
Yesterday I implemented the mod described in my previous comment. I also made the `Rolled` message use a `Vec` and got it so that the web front-end uses that Vec. I chose to make it so that only zero, one or two dice will be displayed on the felt of the graphical interface. Generalizing to N dice would be trivial, but just adding two (since we already supported zero or one) was even easier.

With that groundwork already done, I'll now add the actual game. Since I had a night to sleep on it, I've decided to threat  "x or better" as an Omaha variant, like we do with Binglaha.  Another, more complicated, implementation would be to change how the high-low option is implemented, which would make it so we could play x-or-better in things other than Omaha, but even though that's more general and not that hard to do, it would still take more time.
